io.github.JakubMelka.Pdf4qt.yml

```
@@ -0,0 +1,23 @@
+app-id: io.github.JakubMelka.Pdf4qt
+runtime: org.kde.Runtime
+runtime-version: 5.15.22.08
+sdk: org.freedesktop.Sdk
+command: pdf4qt
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=ipc
+  - --device=dri
+modules:
+  - name: openjpeg
+    buildsystem: cmake-ninja
+    sources:
+        type: archive
```

```suggestion
      - type: archive
```

`sources` is an array of objects.